app/src/main/java/com/example/henzoshimada/feeltrip/EditMoodActivity.java

```diff
@@ -129,13 +191,15 @@ public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
     private void addItemsOnEmotionalStateSpinner(){
         emotionalStateSpinner = (Spinner) findViewById(R.id.emotional_state_spinner);
         List<String> emotionalStateList = new ArrayList<>();
-        emotionalStateList.add("Anger");
-        emotionalStateList.add("Confusion");
-        emotionalStateList.add("Disgust");
-        emotionalStateList.add("Fear");
-        emotionalStateList.add("Happiness");
-        emotionalStateList.add("Sadness");
-        emotionalStateList.add("Shame");
+        emotionalStateList.add("Angry " + getEmojiByUnicode(emojiUnicode("Angry")));
+        emotionalStateList.add("Confused " + getEmojiByUnicode(emojiUnicode("Confused")));
+        emotionalStateList.add("Disgusted " + getEmojiByUnicode(emojiUnicode("Disgusted")));
+        emotionalStateList.add("Fearful " + getEmojiByUnicode(emojiUnicode("Fearful")));
+        emotionalStateList.add("Happy " + getEmojiByUnicode(emojiUnicode("Happy")));
+        emotionalStateList.add("Sad " + getEmojiByUnicode(emojiUnicode("Sad")));
+        emotionalStateList.add("Shameful " + getEmojiByUnicode(emojiUnicode("Shameful")));
+        emotionalStateList.add("Cool " + getEmojiByUnicode(emojiUnicode("Cool")));
+
```

Comment:
we need to use images for custom emojis when we apply themes instead of unicode emojis